hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMultiTargetMessageTask.java

```diff
@@ -74,6 +75,17 @@ protected CompletableFuture<Object> processInternal() {
         return finalResult;
     }
 
+    protected void rethrowInReduce(Map<Member, Object> map, boolean ignoreMemberLeftException) throws Throwable {
+        for (Object item : map.values()) {
+            if (item instanceof Throwable) {
+                if (ignoreMemberLeftException && (item instanceof MemberLeftException)) {
```

Comment:
We should also ignore `TargetNotMemberException` here, it's essentially the same error, one is thrown if the member is missing before the operation is sent, and the other if member leaves after the op was sent. Many `reduce` implementations do the same.

I even think that we should create a common superinterface for these two, because they are almost always checked together, and where they are not, I think it's a mistake, though I'm not sure in all places about this.

Also rename the `ignoreMemberLeftException` to something like `ignoreMissingMemberErrors`

Reply:
Moved to ```ExceptionUtil``` and added ```TargetNotMemberException``` to ignored list.